includes/ECommerce.php

```diff
@@ -106,8 +106,8 @@ public function __construct( Container $container ) {
 		add_filter( 'manage_edit-page_sortable_columns', array( $this, 'sortable_columns' ) );
 		add_action( 'wp_login', array( $this, 'show_store_setup' ) );
 		add_action( 'auth_cookie_expired', array( $this, 'show_store_setup' ) );
-
-		add_action('admin_enqueue_scripts', array( $this, 'set_wpnav_collapse_setting'));			
+		add_action('admin_head', array( $this, 'hide_wp_pointer_with_css' ) );
+		add_action('admin_enqueue_scripts', array( $this, 'set_wpnav_collapse_setting'));
 
 		if ( ( $container->plugin()->id === 'bluehost' && ( $canAccessGlobalCTB || $hasYithExtended ) ) || ( $container->plugin()->id === 'hostgator' && $hasYithExtended ) ) {
```

Comment:
No Yoda here?

Reply:
I haven't made any changes to the existing code.